app/fullService/api/getAccount.ts

```diff
@@ -12,8 +15,53 @@ type GetAccountResult = {
   account: Account;
 };
 
+export function convertAccountV2(
+  accountStatus: AccountStatus,
+  accountSecrets: AccountSecretsV2
+): GetAccountResult {
+  return {
+    account: {
+      accountHeight: accountStatus.account.nextBlockIndex,
+      accountId: accountStatus.account.id,
+      accountKey: accountSecrets.accountKey,
+      entropy: accountSecrets.entropy,
+      firstBlockIndex: accountStatus.account.firstBlockIndex,
+      keyDerivationVersion: accountSecrets.keyDerivationVersion,
+      mainAddress: accountStatus.account.mainAddress,
+      name: accountStatus.account.name,
+      nextSubaddressIndex: accountStatus.account.nextSubaddressIndex,
+      object: 'account',
+      recoveryMode: accountStatus.account.recoveryMode,
+    },
+  };
+}
+
+export const isAccountSynced = async ({ accountId }: GetAccountParams): Promise<boolean> => {
+  const { result, error }: AxiosFullServiceResponse<AccountStatus> = await axiosFullService(
+    GET_ACCOUNT_METHOD,
```

Comment:
see comments in `App.tsx` about wrapping this into a `syncAccount` function that updates the state when `isAccountSynced` is false